microsvcs/gpt_generation/README.md

```diff
@@ -0,0 +1,85 @@
+# api quick reference sheet
+## POST /api/scenario
+*Debating on a GET? maybe with query instead?* 
+
+creates a new scenario for the user. 
+
+@params req.body.uid - required, user id, currently email <br>
+@params req.body.topic - optional, a selected or predefined topic to be used <br>
+
+Sample request body:
+```
+{
+    "uid":"sammyho@email.com", //required
+    "topic":"Diversity" //optional 
+}
+```
+
+Sample success response: 
+```
+{
+    "message": "Scenario Generated",
+    "success": true,
+    "content": {
+        "scenario": "I was facilitating a diversity and inclusion training session on the topic of gender inclusivity. During the session, there was a tense moment when a participant made a derogatory remark about non-binary individuals, which made some other participants uncomfortable.",
+        "options": {
+            "passive": "You should have ignored the remark and continued with the training, hoping that the discomfort would fade away.",
+            "reactive": "You should have addressed the remark immediately, pointing out its inappropriateness and its impact on creating a safe and inclusive environment for all participants.",
+            "proactive": "You should have set clear ground rules at the beginning of the training, emphasizing the importance of respect and inclusivity, to prevent such remarks from being made in the first place.",
+            "active": "You should have used the moment as a teaching opportunity, engaging the group in a discussion about gender diversity, asking the participant to elaborate on their remark and guiding them towards understanding and empathy."
+        }
+    }
+}
+
+```
+---
+## POST /api/scenario/end
```

Comment:
Can use PUT with the same route as above, with scenario id as part of the request body

Reply:
corrected